diff -up breeze-5.26.3/CMakeLists.txt.omv~ breeze-5.26.3/CMakeLists.txt
--- breeze-5.26.3/CMakeLists.txt.omv~	2022-11-08 19:25:51.088950058 +0100
+++ breeze-5.26.3/CMakeLists.txt	2022-11-08 19:25:57.845672491 +0100
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.16)
 project(breeze)
-set(PROJECT_VERSION "5.26.2")
+set(PROJECT_VERSION "5.26.3")
 set(PROJECT_VERSION_MAJOR 5)
 
 set(KF5_MIN_VERSION "5.98.0")
